cfme/tests/generic_objects/test_instances.py

```diff
@@ -242,8 +242,10 @@ def test_generic_objects_tag_rest(appliance, generic_object, tags):
         test_flag: rest
 
     Polarion:
-        assignee: anikifor
+        assignee: pvala
         initialEstimate: 1/4h
+        casecomponent: Rest
+        caseimportance: high
```

Comment:
Required:
please remove this tag... As per my info `caseimportance` should take care by `tire`. Please add respective `tire`. It will be great.

Reply:
@digitronik 'tier' is caselevel, that is different than caseimportance. caseimportance needs to be set. 

caselevel refers to component, integrations, system, or acceptance.